<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnotationDataDisplayWidget</class>
 <widget class="QWidget" name="AnnotationDataDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Annotation Data Display</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutModeControl">
     <item>
      <widget class="QLabel" name="labelDisplayMode">
       <property name="text">
        <string>表示モード:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxDisplayMode">
       <property name="toolTip">
        <string>表示モードを選択（読み取り専用/編集可能）</string>
       </property>
       <item>
        <property name="text">
         <string>読み取り専用</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>編集可能</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerModeControl">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTags">
     <property name="title">
      <string>タグ</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutTags">
      <item>
       <widget class="QTextEdit" name="textEditTags">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>タグが表示されます</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCaption">
     <property name="title">
      <string>キャプション</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutCaption">
      <item>
       <widget class="QTextEdit" name="textEditCaption">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="placeholderText">
         <string>キャプションが表示されます</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxScores">
     <property name="title">
      <string>品質スコア</string>
     </property>
     <layout class="QGridLayout" name="gridLayoutScores">
      <item row="0" column="0">
       <widget class="QLabel" name="labelAestheticScore">
        <property name="text">
         <string>Aesthetic:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelAestheticValue">
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelMUSIQScore">
        <property name="text">
         <string>MUSIQ:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelMUSIQValue">
        <property name="text">
         <string>-</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelOverallScore">
        <property name="text">
         <string>総合スコア:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="sliderOverallScore">
        <property name="toolTip">
         <string>総合スコア</string>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="readOnly" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelOverallValue">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxHistory">
     <property name="title">
      <string>履歴・バージョン</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutHistory">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutHistoryControl">
        <item>
         <widget class="QLabel" name="labelHistoryVersion">
          <property name="text">
           <string>バージョン:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxHistoryVersion">
          <property name="toolTip">
           <string>アノテーション履歴のバージョンを選択</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerHistory">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRevertVersion">
          <property name="text">
           <string>復元</string>
          </property>
          <property name="toolTip">
           <string>選択したバージョンに復元</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditHistoryDetails">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>履歴詳細が表示されます</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDataActions">
     <item>
      <spacer name="horizontalSpacerDataActions">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClearData">
       <property name="text">
        <string>クリア</string>
       </property>
       <property name="toolTip">
        <string>表示データをクリア</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExportData">
       <property name="text">
        <string>エクスポート</string>
       </property>
       <property name="toolTip">
        <string>データをファイルにエクスポート</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>